srcpkgs/hg-evolve/template

```diff
@@ -15,7 +15,7 @@ homepage="https://www.mercurial-scm.org/doc/evolution/"
 changelog="https://repo.mercurial-scm.org/evolve/file/tip/CHANGELOG"
```

Comment:
```suggestion
changelog="https://repo.mercurial-scm.org/evolve/raw-file/tip/CHANGELOG"
```

Reply:
Does this open in the $BROWSER for you with `xchangelog`? It prompts me to download the file.

Reply:
> Content-Type: application/binary

ugh, looks like [misconfig on their end](https://forum.tuxfamily.org/topic/633/mercurial-webserver-changed-behaviour-of-raw-files/)